pwalvarado/counting_vowels.rb

```diff
@@ -0,0 +1,54 @@
+require 'benchmark'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.